app/services/case_finder_service.rb

```diff
@@ -111,15 +111,29 @@ def open_cases_scope
   def erasable_cases_scope
     retention_cases_scope.where(
       retention_schedule: { 
-        state: [:to_be_anonymised]
+        state: [:to_be_anonymised],
```

Comment:
I suggest using the (automatically generated) state constants everywhere. In this case this `:to_be_anonymised` would become `RetentionSchedule::STATE_TO_BE_ANONYMISED`.

It makes the code a lot less fragile, because if for any reason in the state machine we change the name of a state, the constants will break, whereas the symbols or strings depending on the code, might not break and continue working (just not as expected).

I think the tests are fine to use symbols or strings tho.

Reply:
Yep this makes perfect sense will amend to do this - this is the first time I've used aasm so I wasn't aware it made these constants available until I read your most recent PR 👍